pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -49,6 +49,11 @@ lib.makeOverridable (
 , propagatedUserEnvPkgs ? []
 , buildFlags ? []
 , passthru ? {}
+# bundler expects gems to be stored in the cache directory for certain actions
+# such as `bundler install --redownload`.
+# At the cost of increasing the store size, you can keep the gems to have closer
+# alignment with what Bundler expects.
+, keepGemCache ? false
```

Comment:
@zimbatm actually decided to keep it default false largely because I rather see this merged sooner
(trying to upstream my personal fixes).

Reply:
Sounds good